src/matter/cluster/server/OperationalCredentialsServer.ts

```diff
@@ -109,16 +110,18 @@ export const OperationalCredentialsClusterHandler: (conf: OperationalCredentials
     removeFabric: async ({ request: {fabricIndex}, session }) => {
         const device = session.getContext();
 
-        try {
+        const status = tryCatch(() => {
```

Comment:
Nit: indentation. the } should not be on the base line
const status = tryCatch(() => {
        device.removeFabric(fabricIndex);
        return OperationalCertStatus.Success;
    }, FabricNotFoundError,  OperationalCertStatus.InvalidFabricIndex);